os/board/esp32_devkit/src/spiram.c

```diff
@@ -164,7 +161,7 @@ size_t esp_spiram_get_size()
  Before flushing the cache, if psram is enabled as a memory-mapped thing, we need to write back the data in the cache to the psram first,
  otherwise it will get lost. For now, we just read 64/128K of random PSRAM memory to do this.
 */
-void IRAM_ATTR esp_spiram_writeback_cache()
+void IRAM_ATTR esp_spiram_writeback_cache(void)
```

Comment:
Actually, this kind of changes is fixing violations of coding rules. Could you move to 2nd commit?

Reply:
updated, similar modifications moved to 2nd commit